aerogear/unifiedpush-operator
# feat(Monitor) : Add graphana dashboard for UPS Server

## Motivation
https://issues.jboss.org/browse/INTLY-1444

## What
- add graphana dashboard for UPS Server
 
## Verification Steps
Check the dashboard here: https://grafana-route-middleware-monitoring.apps.london-4bc4.openshiftworkshop.com/d/_cxgpLnZx/unifiedpush-server?refresh=10s&orgId=1

## Checklist:


## Progress


## Additional Notes

<img width="1662" alt="Screenshot 2019-07-11 at 06 26 45" src="https://user-images.githubusercontent.com/7708031/61039345-ef85d400-a3a4-11e9-90ca-e2eace4c2173.png">

NOTE: After the metrics be exported we will come back here to check if all are working as we expected. 
See https://issues.jboss.org/browse/INTLY-2668
